morula.yml

```diff
@@ -1,5 +1,5 @@
 before-all: ""
-after-all: "documentation"
+after-all: ""
 always: ""
-never: ""
+never: ".github"
```

Comment:
Directories starting with a dot are (or at least should be) always ignored: https://github.com/Originate/morula/blob/master/features/ignore.feature#L11-L20

Are you using Morula 0.1?